src/structures/Interaction.js

```diff
@@ -173,6 +173,14 @@ class Interaction extends Base {
     return InteractionTypes[this.type] === InteractionTypes.APPLICATION_COMMAND && typeof this.targetId !== 'undefined';
   }
 
+  /**
+   * Indicates whether this interaction is a {@link ModalSubmitInteraction}
+   * @returns {boolean}
+   */
+  isModalSubmission() {
```

Comment:
b621d1863926bd6663fca2c1291a3512beca883a from the v14 PR reverts the name back to `isModalSubmit`.

Reply:
God damnit @suneettipirneni